apps/user/v1/views.py

```diff
@@ -57,29 +76,19 @@ def me(self, request):
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
         return Response(verify_token(request.data["token"]), status=status.HTTP_200_OK)
 
-    @action(
-        methods=["POST"],
-        detail=False,
-        url_path="pass-reset",
-        url_name="pass-reset",
-        serializer_class=PasswordResetSerializer,
-    )
+    @action(methods=["POST"], detail=False,url_path="pass-reset",url_name="pass-reset", serializer_class=PasswordResetSerializer)
     def password_reset(self, request):
         try:
             serializer = self.get_serializer(data=request.data)
-            serializer.is_valid(raise_exception=True)
-
+            serializer.is_valid()
             username_or_email = serializer.validated_data["username_or_email"]
             user = get_object_or_404(User, Q(email=username_or_email) | Q(username=username_or_email))
             send_reset_email(user)
-
             return Response(status=status.HTTP_204_NO_CONTENT)
         except Exception:
-            return Response({"error": "ERROR.PASSWORD_RESET"}, status=status.HTTP_400_BAD_REQUEST)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Esto lo cambias por algún motivo? Ya estaba devolviendo un literal de traducción... así tendríamos que cambiar más cosas en el Serializador

Reply:
Esto es que no me sale this field is required(username_email) y para testear lo cambie